enos/enos-scenario-replication.hcl

```diff
@@ -0,0 +1,667 @@
+scenario "replication" {
```

Comment:
Can we add a comment with more context than just replication? I think a `description` field in a `scenario` that we can use in enos to automatically tell us that additional information would be great. I created QT-456 for it just now. Until then, a bit more context up front as a comment might be nice.

Reply:
done